prerequisites.md

```diff
@@ -30,58 +31,74 @@ WSL has our preference, with Git for Windows acting as a backup-option. Guide fo
 
 ## Generating a keypair in Windows PowerShell
 
-1. In PowerShell, type the following commands (press ENTER after each ```powershell
+1. In PowerShell, type the following commands (press ENTER after each command)
+
+```powershell
 mkdir %userprofile%/.ssh
 cd %userprofile%/.ssh
 ssh-keygen -t rsa -C "your_email@example.com"
 ```
+
 2. Follow the instructions. Note that you will have to set a password.
 3. It will save the key in the folder %userprofile%/.ssh. Use a file explorer to navigate to this folder.
 4. Mail the [public key](##glossary) file to the trainer.
 
 The rest of the instructions are the same as those under [Using WSL](#usewsl)
 
-
 <a name="getwsl"></a>
 
 ## Getting WSL
+
 1. Open PowerShell (WIN+X, A), and run the following command:
+
 ```powershell
 Enable-WindowsOptionalFeature -Online -Featureame Microsoft-Windows-Subsystem-Linux
```

Comment:
looks like a typo in the command `-Featureame`

Reply:
Is indeed some kind of typo, fixed.